src/main/java/io/stargate/sgv2/jsonapi/metrics/MicrometerConfiguration.java

```diff
@@ -1,17 +1,16 @@
-package io.stargate.sgv2.jsonapi.api.v1.metrics;
+package io.stargate.sgv2.jsonapi.metrics;
```

Comment:
ok to move this one, the config classes to leave are the ones with @ConfigMapping(